lib/secure_headers/railtie.rb

```diff
@@ -7,7 +7,7 @@ class Railtie < Rails::Railtie
                              'X-Permitted-Cross-Domain-Policies', 'X-Download-Options',
                              'X-Content-Type-Options', 'Strict-Transport-Security',
                              'Content-Security-Policy', 'Content-Security-Policy-Report-Only',
-                             'Public-Key-Pins', 'Public-Key-Pins-Report-Only']
+                             'Public-Key-Pins', 'Public-Key-Pins-Report-Only, Referrer-Policy']
```

Comment:
Is this supposed to be two separate strings instead of continuing the PKP-RO string?
```
'Public-Key-Pins-Report-Only', 'Referrer-Policy'
```